controllers/autoneg.go

```diff
@@ -377,8 +398,16 @@ func validateOldConfig(cfg OldAutonegConfig) error {
 	return nil
 }
 
-func validateNewConfig(cfg AutonegConfig) error {
-	// do additional validation
+func validateNewConfig(config AutonegConfig) error {
+	for _, cfgs := range config.BackendServices {
+		for _, cfg := range cfgs {
+			if cfg.InitialCapacity != nil {
+				if *cfg.InitialCapacity < 0 || *cfg.InitialCapacity > 100 {
+					return errors.New("initial_capacity must be between 0 and 100 inclusive")
```

Comment:
I thought about using fmt.Errorf here and including error details (such as the name of the backend or port), or possibly a custom error type.. but in practice, callers are likely to just check if `err != nil`

Reply:
It'd be good to have the details of the error, since there can be many services on a cluster operated by different teams but with one autoneg controller.

Reply:
added the backend name as well as the parsed value to facilitate debugging